x-pack/plugins/session_view/public/components/tty_player/hooks.ts

```diff
@@ -234,7 +234,7 @@ export const useXtermPlayer = ({
           // there is some random race condition with the jump to feature that causes these calls to error out.
         }
       } else {
-        linesToPrint = lines.slice(lineNumber, lineNumber + TTY_LINES_PER_FRAME);
+        linesToPrint = lines.slice(lineNumber, lineNumber + 1);
```

Comment:
interesting

Reply:
yea, it was an optimization in an attempt to limit the amount of state changes. caused more problems than solved..

Reply:
Got it!

If rendering shows to be a problem for larger sections, we can think about other solutions, either using Throttle or [requestAnimationFrame](https://css-tricks.com/using-requestanimationframe-with-react-hooks/), then we can make playback faster, but internally we are optimizing when to render.